tests/network/vmi_slirp_interface.go

```diff
@@ -72,101 +73,112 @@ var _ = SIGDescribe("[Serial]Slirp Networking", func() {
 		currentConfiguration = kv.Spec.Configuration
 	}
 
-	tests.BeforeAll(func() {
+	BeforeEach(func() {
+		tests.BeforeTestCleanup()
+
 		virtClient, err = kubecli.GetKubevirtClient()
 		util.PanicOnError(err)
 
 		kv := util.GetCurrentKv(virtClient)
 		currentConfiguration = kv.Spec.Configuration
+	})
+
+	Context("slirp is not the default interface", func() {
+		BeforeEach(func() {
+			setSlirpEnabled(true)
+			ports = []v1.Port{{Name: "http", Port: 80}}
 
-		setSlirpEnabled(true)
-		ports := []v1.Port{{Name: "http", Port: 80}}
-		genericVmi = tests.NewRandomVMIWithSlirpInterfaceEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n", ports)
-		deadbeafVmi = tests.NewRandomVMIWithSlirpInterfaceEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n", ports)
-		deadbeafVmi.Spec.Domain.Devices.Interfaces[0].MacAddress = "de:ad:00:00:be:af"
+			vmi = tests.NewRandomVMIWithSlirpInterfaceEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n", ports)
```

Comment:
The vmi has to be created in the fixture because we are altering the object in each table entry [here](https://github.com/kubevirt/kubevirt/blob/2961171acb1c8cb61a176d9a4c20164efb9145df/tests/network/vmi_slirp_interface.go#L99).
That's why I thought making the base object  in the setup, and then alter and create in the fixture make sense.

Now, even though I changed it to use the vmi-ref instead of the mac address boolean, I still think it is better to keep it in the fixture and avoid creating two vmis in the beofreEach where we only need one in each entry.